Assets/Scripts/CineMachineCamara.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Cinemachine;
+
+public class CineMachineCamara : MonoBehaviour
+{
+    public static CineMachineCamara Instance;
+    private CinemachineVirtualCamera _cinemachineVirtualCamera;
+    private CinemachineBasicMultiChannelPerlin _cinemachineBasicMultichannelPerlin;
+    private float timeMovement;
+    private float initialMovementTime;
+    private float initialIntencity;
```

Comment:
Use guidelines.

All should be initialized and start with `_` and `public`must be `[SerializeField] private`